sdk/core/Azure.Core.Experimental/tests/perf/DynamicReadingBenchmark.cs

```diff
@@ -13,9 +14,10 @@ namespace Azure.Data.AppConfiguration.Performance
     public class DynamicReadingBenchmark
     {
         private static string _json = "{\"a\":{\"b\":5}}";
+        private static BinaryData _binaryData = new BinaryData(_json);
 
         private static dynamic _expandoObject = new ExpandoObject();
-        private static dynamic _jsonData = JsonData.FromString(_json);
+        private static dynamic _jsonData = _binaryData.ToDynamic();
```

Comment:
Cool, that makes sense.  We could have it be ToDynamicJson() ... although that makes me wonder if we should change the typename to DynamicJson rather than JsonData ... is there a strong need for Data to be in the name?

Reply:
BinaryData has a method `FromObjectAsJson`, so maybe we call this one `ToDynamicFromJson`, but such name is on the longish side. I would prefer something shorter. We could also take a parameter that selects between formats that we support. For now, it would be JSON but in the future we could support XML, protobuff, etc. 

I don't think calling the type DynamicJson would help. This type name is not visible when `ToDynamic` is called. 

Lastly, that would close the door to reusing this type for other formats besides JSON.

Reply:
Per offline conversation, we will leave this as-is for now.